src/wordpress.js

```diff
@@ -6,17 +6,24 @@ const fetch = require('node-fetch');
 const flatcache = require('flat-cache');
 const { DateTime } = require('luxon');
 
-const AMO_BLOG_BASE_URL = 'https://addons.mozilla.org/blog';
-const DEFAULT_WORDPRESS_BASE_URL = 'https://mozamo.wpengine.com';
+const AMO_BLOG_BASE_URL =
+  process.env.AMO_BLOG_BASE_URL || 'https://addons.mozilla.org/blog';
+const WORDPRESS_BASE_URL =
+  process.env.WORDPRESS_BASE_URL || 'https://mozamo.wpengine.com';
 
 async function getNumPages(endPoint) {
   const result = await fetch(endPoint, { method: 'HEAD' });
   return result.headers.get('x-wp-totalpages') || 1;
 }
 
-const fixInternalURLs = (content) => {
+const fixInternalURLs = (content, { baseURL = WORDPRESS_BASE_URL } = {}) => {
+  if (process.env.DONT_FIX_INTERNAL_URLS === '1') {
+    return content;
+  }
+
   return content.replace(
-    /https?:\/\/mozamo\.wpengine\.com/g,
+    // Make sure the URL pattern accepts both http:// and https://.
+    new RegExp(baseURL.replace(/^https?:/, 'https?:'), 'g'),
```

Comment:
Does this need to be a modified constant? It is probably easier to understand if it's a regex or a constant that's a regex.

Reply:
The constant is used elsewhere I think. In this case, it's used to create a regex because I didn't want to manually escape the slashes in the base URL and, as a bonus, it allows me to slightly update the protocol to handle both http and https.

Reply:
It's covered by tests so I think we don't need to worry about that right now. I think it's ok to reuse a constant even if we modify it. The constant itself doesn't have to be a regex.